controller/workitemtype.go

```diff
@@ -95,9 +133,14 @@ func (c *WorkitemtypeController) ListSourceLinkTypes(ctx *app.ListSourceLinkType
 
 // ListTargetLinkTypes runs the list-target-link-types action.
 func (c *WorkitemtypeController) ListTargetLinkTypes(ctx *app.ListTargetLinkTypesWorkitemtypeContext) error {
+	spaceID, err := uuid.FromString(ctx.ID)
+	if err != nil {
+		return errors.NewNotFoundError("spaceID", ctx.ID)
+	}
+
 	return application.Transactional(c.db, func(appl application.Application) error {
 		// Test that work item type exists
-		_, err := appl.WorkItemTypes().Load(ctx.Context, ctx.WitID)
+		_, err := appl.WorkItemTypes().Load(ctx.Context, spaceID, ctx.WitID)
```

Comment:
the use of space_id is most likely optimized away by the query planner. It sees the witid, the primary key and thinks; we'll that is a unique index so I'm only expecting to get 0.1 result back, any attempt at a space index would be 0.n, so cost wise I'll go for 0 to max 1. Space id just becomes, maybe a odd safety check that the found wit belong to the expected space.